feature-22401 (#110)
ArrowSphere/public-api-client: doc/billing-statements.md

# Billing statements client

## General information

> Billing statements are the documents issued prior to the issue of the local Invoice.

Reviewer: ```suggestion
Billing statements are the documents issued prior to the issue of the local invoice.
```

> This client gives a uniform access to all the billing information before their transformation 

Reviewer: ```suggestion
This client gives an uniform access to all the billing information before their transformation 
```

> into the invoice that'll fit the local country's legislation.

Reviewer: ```suggestion
into the invoice that will fit the local country's legislation.
```

The billing statement will mainly contain transactional data.
The Invoices will have local information such as tax rates and full coordinates 
of the transacting entities.

Billing statements are composed of two elements:
- The header
- The line

> Billing statements & their lines are technically referenced by their "reference" field,

Reviewer: ```suggestion
Billing statements and their lines are technically referenced by their `reference` field,
```
